assets/css/main.css

```diff
@@ -0,0 +1,516 @@
+:root {
+  --nw-red: #f72210;
+}
+body {
+  background-color: #f4f5f6;
+  font-family: "Roboto Condensed", sans-serif;
+}
+h5 > a:hover {
+  color: inherit;
+  text-decoration: underline;
+  text-decoration-color: var(--nw-red);
+}
+
+/* BOOTSTRAP OVERRULE */
+
+a {
+  color: inherit;
+}
+.container-fluid {
+  max-width: 1310px;
+  background-color: #fff;
+}
+
+/* HEADER */
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 10;
+  background-color: var(--nw-red);
+  height: 50px;
+  padding-left: 20px;
+  padding-right: 10px;
+}
+header a:hover {
+  text-decoration: none;
+  color: inherit;
+}
+#meteo {
+  display: none;
+}
+#title {
+  font-size: 2em;
+  font-family: "Rokkitt", serif;
+  line-height: 1.5;
+  font-weight: bold;
+}
+#login {
+  margin-right: 10px;
+}
+#login,
+#subscribe-btn {
+  font-size: 0.8em;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+#subscribe-btn {
+  background-color: #ffe400;
+  border-radius: 99em;
+  padding: 1px 10px;
+  white-space: nowrap;
+}
+#hamburger {
+  font-size: 1em;
+  color: #fff;
+  margin-left: 0.5em;
+}
+
+/* NAV */
+
+#top > *:not(header) {
+  display: none;
+}
+#nav-1 > span {
+  color: #999;
+  font-size: 0.5em;
+  font-weight: normal;
+  border-right: 1px solid #ddd;
+}
+#nav-1,
+#nav-2 {
+  font-weight: bold;
+}
+#nav-1 > a:hover,
+#nav-2 > a:hover {
+  color: var(--nw-red);
+  text-decoration: none;
+}
+#nav-1 {
+  font-size: 1.1em;
+  border-bottom: 2px solid #eee;
+}
+#nav-1 input {
+  max-width: 3em;
+  border: none;
+}
+#nav-2 {
+  border-bottom: 1px solid #ccc;
+}
+#nav-2 > * {
+  text-transform: uppercase;
+  line-height: 0;
+  padding-top: 0.5em;
+}
+#nav-2 > a {
+  color: #999;
+}
+#pretty-bar-1 {
+  height: 14px;
+  background: linear-gradient(to right, #ddd, #fff);
+}
+#pretty-bar-2 {
+  height: 6px;
+  border-bottom: 2px solid #eee;
+}
+
+/* NEWS */
+
+#news {
+  margin-top: 60px;
+}
+#news h2 {
+  color: #999;
+  text-transform: uppercase;
+  font-size: 1em;
+  font-weight: bold;
+}
+#news h5 {
+  font-size: 1.5em;
+}
+#my-turn > div > div:first-child,
+#culture-travel > div > div:first-child {
+  padding-left: 0;
+}
+#my-turn > div > div:last-child,
+#culture-travel > div > div:last-child {
+  padding-right: 0;
+}
+#my-turn .card-body,
+#culture-travel .card-body {
+  color: #999;
+  font-weight: bold;
+}
+#culture-travel .card-text {
+  padding: 2px 5px !important;
+  color: #333;
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_